ci/provisioning/post_provision_config_common_functions.sh

```diff
@@ -190,7 +190,7 @@ set_local_repo() {
     version=${version%%.*}
     if [ "$repo_server" = "artifactory" ] &&
        { [[ $(pr_repos) = *daos@PR-* ]] || [ -z "$(rpm_test_version)" ]; } &&
-       [[ ! ${CHANGE_TARGET:-$BRANCH_NAME} =~ ^[-0-9A-Za-z]+-testing ]]; then
+       [[ ! ${CHANGE_TARGET:-$BRANCH_NAME} =~ ^[.-0-9A-Za-z]+-testing ]]; then
```

Comment:
My regex is fuzzy,  but doesn't `[.-0-9A-Za-z]+` simplify to `.+`?
```suggestion
       [[ ! ${CHANGE_TARGET:-$BRANCH_NAME} =~ ^.+-testing ]]; then
```

Reply:
Not really.  `[.-0-9A-Za-z]` is still a subset of `.`.  I tend to try to be conservative with REs just to avoid unintended matches.

Reply:
Doesn't `.` mean any character?

Reply:
No, we don't want to match '-' for example.

Reply:
> Doesn't `.` mean any character?

Not inside of a class, no.